README.md

```diff
@@ -335,6 +322,29 @@ tokenName: 'token',
 tokenRoot: false,
 
 
+// Refresh Token Options
+// =====================
+
+// Option to turn refresh tokens On/Off
+useRefreshToken: false,
+// The option to enable/disable the automatic refresh of Auth tokens using Refresh Tokens
+autoUpdateToken: true,
+// The the property from which to get the refresh token after a successful token refresh
+refreshTokenProp: 'refresh_token',
+// Prepended to the `refreshTokenName` when kept in storage (nothing to do with refresh token responses)
+refreshTokenPrefix: 'aurelia',
```

Comment:
Sure. But it's just the prefix in the option. Post.fixes the need to be different






On Fri, Apr 1, 2016 at 1:33 PM -0700, "Adam Willden" <notifications@github.com> wrote:





> @@ -335,6 +322,29 @@ tokenName: 'token',
>  tokenRoot: false,
>
>
> +// Refresh Token Options
> +// =====================
> +
> +// Option to turn refresh tokens On/Off
> +useRefreshToken: false,
> +// The option to enable/disable the automatic refresh of Auth tokens using Refresh Tokens
> +autoUpdateToken: true,
> +// The the property from which to get the refresh token after a successful token refresh
> +refreshTokenProp: 'refresh_token',
> +// Prepended to the `refreshTokenName` when kept in storage (nothing to do with refresh token responses)
> +refreshTokenPrefix: 'aurelia',

I haven't looked into it but I think both access and refresh tokens need their own storage locations. Perhaps @alfkonee  can comment here to confirm either way.

---
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/SpoonX/aurelia-authentication/pull/86/files/e8c083f412d22fbbbb3f9023f8b81aacf267a4ad#r58263498